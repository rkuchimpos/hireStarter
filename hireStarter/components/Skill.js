import React from 'react';
import { Button, StyleSheet, Text, View } from 'react-native';

class Skill extends React.Component {
  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.skill}>{this.props.skill}</Text>
      </View>
    );
  }
}
const styles = StyleSheet.create({
  container: {
    backgroundColor: '#FF5F6D',
    borderRadius: 20,
    padding: 8,
    paddingLeft: 10,
    paddingRight: 10,
    alignSelf: 'flex-start',
  },
  skill: {
    fontSize: 14,
    color: '#fff',
  }
});


export default Skill;
