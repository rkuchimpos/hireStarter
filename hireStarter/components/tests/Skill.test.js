import React from 'react';
import renderer from 'react-test-renderer';
import Skill from '../Skill';

it('Skill renders correctly', () => {
  const tree = renderer
    .create(<Skill skill={'Test'}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
