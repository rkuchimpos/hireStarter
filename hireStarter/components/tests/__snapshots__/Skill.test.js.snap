// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skill renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "flex-start",
      "backgroundColor": "#4A00E0",
      "borderRadius": 20,
      "padding": 8,
      "paddingLeft": 10,
      "paddingRight": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 14,
      }
    }
  >
    Test
  </Text>
</View>
`;
